### ISSUE TITLE
I have different result with loaded model

### ISSUE THREAD

**Issue by JiHyeoNEE** [keras-team/autokeras#1222]
I saved the model like this,

```
final_model = model.export_model()
final_model.predict(y_test)
final_model.save('my_model.h5')

```

and I loaded the model like this
```
custom = ak.CUSTOM_OBJECTS
custom['Normalization'] = tf.keras.layers.experimental.prerprocessing.Normalization
loaded_model = load_model('my_model.h5', custom_objects=custom)
loaded_model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])
loaded_model.predict(y_test)
```

I have different results between final_model.predict() and loaded_model.predict() with the same dataset.
The accuracy of former one is  about 86% but later one is 48%.
Also all the predicted results of loaded_model are just 1. 
Please let me know why it is happened and how can I restore the same results.

 - Python: 3.7.7
 - autokeras: 1.0.2
 - keras-tuner: 1.0.1
 - scikit-learn: 0.22.1
 - numpy: 1.18.2
 - pandas: 1.0.3
 - tensorflow: 2.2.0
- tensorflow-gpu : 2.1.0

Thank you
